### DISCUSSION TITLE
Is there a way to point Act at a local Action repo, not just workflow?

### DISCUSSION THREAD

**Discussion by mcascone** [nektos/act#1873] [Q&A]
I am developing a GitHub Action and would love to be able to have Act pull the Action called by the Workflow from my local Git repo instead of from Github.com. 

It looks like there's _almost_ a way to do this but the docs aren't specific on it. Has anyone figured it out?

I commented in an Issue for the same request here: https://github.com/nektos/act/issues/1579#issuecomment-1574290406

**Comment by shubhbapna** [comment]
Hey I had a documented a workaround here to use local workflows: https://github.com/kiegroup/mock-github/discussions/64#discussioncomment-5727505

Since https://github.com/nektos/act/pull/1784 has been merged and released in 0.2.46, you should be able to override `GITHUB_SERVER_URL` and set it to the location of the parent directory of your local git repo. Hope this helps!

**Comment by ChristopherHX** [comment]
Via an additional layer of https://github.com/nektos/act/pull/1913 it should be possible to use a local folder as the src of an action.

E.g. ignore the Fetch call if a local repo mapping is in a map and create a tar archive from the storage folder for GetTarArchive.

Disclaimer this interface has not been accepted yet and not all changes are uploaded for review.
